lib/esm_cache.py

```diff
@@ -18,6 +19,12 @@ def main(cfg: UAConfig) -> None:
 
 
 if __name__ == "__main__":
+    setup_logging(
+        logging.INFO,
+        logging.DEBUG,
+        defaults.CONFIG_DEFAULTS["log_file"],
+        logger=logging.getLogger(),
```

Comment:
I am mimicking https://github.com/canonical/ubuntu-pro-client/pull/2494/files#diff-6fea9b378e472552634bc16317f26aec387dd558c1754cd545806364476782d0R33 and setting the root logger, but this module uses a specific logger (line 10).

Is there a specific reason to do this, or should we set up the specific logger only?

Reply:
This is right for now. One of the high priority logging revamp tasks @CalvoM has is to make all of our loggers use the `uaclient` logger and then all of these setup_logging calls will be adjusted to configure that instead of the root logger.

But this is doing the right thing until that fix happens.